bitnami/redis/templates/_helpers.tpl

```diff
@@ -222,34 +222,11 @@ Get the password key to be retrieved from Redis&reg; secret.
 {{- end -}}
 {{- end -}}
 
-
-{{/*
-Returns the available value for certain key in an existing secret (if it exists),
-otherwise it generates a random value.
-*/}}
-{{- define "getValueFromSecret" }}
-    {{- $len := (default 16 .Length) | int -}}
-    {{- $obj := (lookup "v1" "Secret" .Namespace .Name).data -}}
-    {{- if $obj }}
-        {{- index $obj .Key | b64dec -}}
-    {{- else -}}
-        {{- randAlphaNum $len -}}
-    {{- end -}}
-{{- end }}
-
 {{/*
 Return Redis&reg; password
 */}}
 {{- define "redis.password" -}}
-{{- if or .Values.auth.enabled .Values.global.redis.password }}
-    {{- if not (empty .Values.global.redis.password) }}
-        {{- .Values.global.redis.password -}}
-    {{- else if not (empty .Values.auth.password) -}}
-        {{- .Values.auth.password -}}
-    {{- else -}}
-        {{- include "getValueFromSecret" (dict "Namespace" (include "common.names.namespace" .) "Name" (include "redis.secretName" .) "Length" 10 "Key" (include "redis.secretPasswordKey" .))  -}}
-    {{- end -}}
-{{- end -}}
+{{ include "common.secrets.passwords.manage" (dict "secret" (include "redis.secretName" .) "key" (include "redis.secretPasswordKey" .) "providedValues" (list "global.redis.password" "auth.password") "length" 10 "skipB64enc" true "skipQuote" true "context" $) }}
```

Comment:
```suggestion
{{- if or .Values.auth.enabled .Values.global.redis.password }}
    {{ include "common.secrets.passwords.manage" (dict "secret" (include "redis.secretName" .) "key" (include "redis.secretPasswordKey" .) "providedValues" (list "global.redis.password" "auth.password") "length" 16 "skipB64enc" true "skipQuote" true "context" $) }}
{{- end }}
```

The password was previously generated under that condition. Apart from that, the default length was 16

Reply:
Thanks for your review. Suggestion applied.
I still believed the default length was 10, though, with the argument when calling the previous function. https://github.com/bitnami/charts/pull/29376/files#diff-ddd93ad4c7925e478912f6a48d16e6ac7d562f34083126c97736d210470f7e04L250

Reply:
You are right @LeoColomb, I didn't notice that and saw that the default length in the function was 16. Accepting the change now